sonar-project.properties

```diff
@@ -0,0 +1,9 @@
+# Required metadata
```

Comment:
As you can see from this page at Step 2 https://pipedrive.atlassian.net/wiki/spaces/SD/pages/12616076/How+to+add+a+project+to+SonarQube

There are some missing props, for ex: 
```
# Path to the source code directory relative to the sonar-project.properties file.
sonar.sources=. 
```
Is this ok?

Reply:
I thought that the required metadata are only:
```
sonar.projectKey=Pipedrive:Green:dogfood
sonar.projectName=Dogfood
```
But yeah, I do agree that the sources should probably be specified

Reply:
Just ran with the sources defined and the output was pretty much the same, I guess I'll specify it since it seems to be best practice but I don't think it is that relevant TBH.